src/sentry/tasks/relay.py

```diff
@@ -99,79 +97,146 @@ def update_config_cache(
         )
 
 
-def schedule_update_config_cache(
-    generate, project_id=None, organization_id=None, public_key=None, update_reason=None
-):
-    """
-    Schedule the `update_config_cache` with debouncing applied.
+@instrumented_task(
+    name="sentry.tasks.relay.build_config_cache",
+    queue="relay_config",
+    acks_late=True,
+    soft_time_limit=30,
+    time_limit=32,
+)
+def build_config_cache(public_key=None, trigger=None, **kwargs):
+    """Build a project config and put it in the Redis cache.
 
-    See documentation of `update_config_cache` for documentation of parameters.
+    This task is the primary way a new project config is computed.
```

Comment:
Not really true, it is when relay asks for a non-existing project config.  
```suggestion
    This task is used to compute missing project configs, it is aggressively
    deduplicated to avoid running duplicate computations and thus should only
    be invoked using :func:`schedule_build_project_config`.  Because of this
    deduplication it is not suitable for re-computing a project config when
    an option changed, use :func:`schedule_invalidate_project_config` for this.
```